computedfields/models.py

```diff
@@ -55,8 +55,12 @@ def __new__(mcs, name, bases, attrs):
                         dependent_fields[k] = depends
         cls = super(ComputedFieldsModelType, mcs).__new__(mcs, name, bases, attrs)
         if name != 'ComputedFieldsModel':
-            cls._computed_fields = computed_fields
-            mcs._computed_models[cls] = dependent_fields or {}
+            if hasattr(cls, '_computed_fields'):
+                cls._computed_fields.update(computed_fields)
+            else:
+                cls._computed_fields = computed_fields
+            if not cls._meta.abstract:
+                mcs._computed_models[cls] = dependent_fields or {}
```

Comment:
To make sense of these lines for me - this basically adds computed field definitions to `cls._computed_fields` over several `cls` constructions, and announcing the final definitions once it hits a non-abstract class, right?

I have not looked up django's dealing of abstract models in their metaclass, do you think there are hidden edge cases, where our metclass handling might break?

Reply:
Yes that's what it does.

I don't know Django internals enough to tell whether there might be hidden edge cases

Reply:
Well the test covers most aspects regarding computed fields, so lets merge it. :smile_cat: